src/base/searchengine.cpp

```diff
@@ -63,7 +63,7 @@ static inline void removePythonScriptIfExists(const QString &scriptPath)
 const QHash<QString, QString> SearchEngine::m_categoryNames = SearchEngine::initializeCategoryNames();
 
 SearchEngine::SearchEngine()
-    : m_updateUrl(QString("http://searchplugins.qbittorrent.org/%1/engines/").arg(Utils::Misc::pythonVersion() >= 3 ? "nova3" : "nova"))
```

Comment:
Oh ok. Basically you want to leave the mechanisms in place for when we introduce a new version number. Makes sense.